### PR TITLE
feat: instrument vertx-web-openapi-router

### PR DESCRIPTION
**Pull request by pk-work**
see #176

### CODE REVIEW

File: vertx-lang-kotlin/pom.xml [hunk 1]
```diff
@@ -183,6 +183,11 @@
       <artifactId>vertx-web-validation</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-openapi-router</artifactId>
```
Comment (tsegismont) [inline vertx-lang-kotlin/pom.xml:188]: You also need to add the `artifactId` to the list in the maven dependency plugin config.

**Review by tsegismont — COMMENTED** [review]
Thanks @pk-work 

Is `vertx-web-openapi` already present?

**Comment by pk-work** [comment]
> Thanks @pk-work
> 
> Is `vertx-web-openapi` already present?

No, but this is good, because in Vert.x 5 it does not longer exist.

File: vertx-lang-kotlin/pom.xml (hunk 1 above)
Comment (pk-work) [reply vertx-lang-kotlin/pom.xml:188]: done

**Review by tsegismont — APPROVED** [review]
LGTM, thanks

**Pull request merged**